changes sleepy pens to be slightly less bad, same with emagged borg cookies, removes chloralhydratedelayed

CrazyClown12 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Up until now the "chloral hydrate" inside of sleepy pens has been its own unique chemical that is literally chloral hydrate but worse (slower). I see no reason why this needs to be the case, as no one seems to use the starter chems in the sleepy pen anyway.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
It's a slight buff to sleeping pens but it is also more consistent. I don't see the need for there to be a special version of an established chemical, especially when the original chemical isn't really that great anyway.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: The chloral hydrate inside of the sleepy pen is no longer slower acting than chloral hydrate made in chemistry.
tweak: The chloral hydrate inside of cookies synthesised by emagged borgs is no longer slower acting than chloral hydrate made in chemistry.
del: Delayed chloral hydrate
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

CrazyClown12 committed:
changes sleepy pens to be slightly less bad

They have special snowflake chloral hydrate that is literally chloral hydrate but worse. Let's make it chloral hydrate but literally chloral hydrate. Maybe then people will actually use the chems in a sleepy pen instead of emptying it and bitching about how bad it is on discord.

CrazyClown12 commented:
First PR don't bully me please

Rowelll commented:
no bully message incoming:

ok, I like it even if I never used it

Barhandar commented:
Oh boy it'll be hilarious if tirizene changes go in and sleepypen turns into basically slightly delayed parapen

necromanceranne commented:
nemvar's waiting on 4d's changes so if those happen, even before nemvar's changes, I'd say it's pretty good for what it does.

Ty-the-Smonk commented:
Im gonna bully you

4dplanner commented:
Why leave in the old reagent? Is it used for anything else?

necromanceranne commented:
I have a feeling it was a relic of the initial nerfs sleepy pen got back when chloral was much, much, much stronger.

nemvar commented:
Sleepy pens barely sleep at all. Even with tirizine buffs, this change will probably be needed. In my testing with new tirizine, I was never asleep for longer than 15 seconds. No time at all if you want to abduct someone.

ExcessiveUseOfCobblestone commented:
> 
> 
> Why leave in the old reagent? Is it used for anything else?

emagged borgs used them else I'd tell him to just remove it.

ATH1909 commented:
> > Why leave in the old reagent? Is it used for anything else?
> 
> emagged borgs used them else I'd tell him to just remove it.

Eh, you might want to keep it in in case the nerf needs to be reverted for some reason.

zxaber commented:
So how long does it take to act now?

CrazyClown12 commented:
> So how long does it take to act now?

From what I gather,  chloralhydrate takes 10 cycles (ticks? idk what the term is for this specific case) to put someone to sleep but Chloralhydratedelayed does the same but it takes 20 cycles. There is also another difference between the two chems. Chloralhydrate will do tox damage after a certain number of ticks, whereas chloralhydratedelayed doesn't. As this requires over 30u, it should not effect the operation of the sleepy pen, as it uses 20u of chloral hydrate (except for edge cases where you get injected by two sleepy pens, but that's extremely unlikely).

>Why leave in the old reagent? Is it used for anything else?

With regards to leaving the reagent in, I thought about removing it then decided against doing that when I saw borgs could use it. However, now that I've thought about it more, we could change it for the borgs as well. Borgs can synthesise chloralhydratedelayed cookies, and to an extent the same arguments apply for using chloral hyrdate. There is also the consideration that chloralhydratedelayed being a purely nonlethal option whereas chloral hydrate can do toxin damage, but that's only really a distinction made in code because of the rare cases and low volumes that chloralhydratedelayed is involved (if you eat more than 3 cookies from an emagged borg you deserve the small amounts of tox damage).


What do people think? Should chloralhydratedelayed be removed entirely?

necromanceranne commented:
Don't see why not simply for consistencies sake.

ExcessiveUseOfCobblestone commented:
I won't lose sleep over it anon

CrazyClown12 committed:
nerfs the cakehat (not a shitpost)

It's pretty retarded that a single candle on a meme hat is treated as an infinite fuel welder

CrazyClown12 committed:
Merge pull request #1 from CrazyClown12/CrazyClown12/cakehat

nerfs the cakehat (not a shitpost)

RaveRadbury commented:
Uh-oh, Cakehat nerf sneaking in!!

ExcessiveUseOfCobblestone commented:
i'm banning you

CrazyClown12 commented:
Oh shit, I didn't mean to do that. I'm new to github and was genuinely practicing committing between 2 forks, didn't mean to push to here. I'm gonna close this and start again lol

ExcessiveUseOfCobblestone commented:
why are you making a new PR to fix a mistake, cringe

CrazyClown12 commented:
> First PR don't bully me please

ExcessiveUseOfCobblestone commented:
if you're going to do it from the browser you forfeit right of no-bully

CrazyClown12 committed:
Revert "Merge pull request #1 from CrazyClown12/CrazyClown12/cakehat"

This reverts commit 4806c57780a3d6f564b2ec58685b3c9005642892, reversing
changes made to 95b82ed3fd48f7ac7e7ce06fd76afb03860b3e8d.

CrazyClown12 commented:
reopening so it may still be broken but I'm trying my best

CrazyClown12 committed:
remove chloralhydratedelayed

I'm gonna fuck this up again

CrazyClown12 committed:
delete chloral hydrate delayed from reagents

CrazyClown12 committed:
Merge pull request #2 from CrazyClown12/CrazyClown12-patch-1

remove chloralhydratedelayed from emagged borgs, replace with chloralhydrate

CrazyClown12 committed:
Merge pull request #3 from CrazyClown12/CrazyClown12-nodelayed

delete chloral hydrate delayed from reagents
